packages/firestore/test/unit/specs/describe_spec.ts

```diff
@@ -135,8 +142,14 @@ export function specTest(
       }
       const mode = usePersistence ? '(Persistence)' : '(Memory)';
       const fullName = `${mode} ${name}`;
-      runner(fullName, () => {
-        return spec.runAsTest(fullName, usePersistence);
+      runner(fullName, async () => {
+        const start = Date.now();
+        await spec.runAsTest(fullName, usePersistence);
+        const end = Date.now();
+        if (tags.indexOf(BENCHMARK_TAG) >= 0) {
+          // tslint:disable-next-line:no-console
+          console.log(`Runtime: ${end - start} ms.`);
+        }
```

Comment:
FWIW- If you wanted to get fancy you could play with integrating console.time() / console.profile() to get higher-precision timings and to automatically collect profiles.  Though they're non-standard. So in theory it could break our tests on some browsers.

Reply:
I am thinking that high profile timers run in a headless Chrome instance might give us more of an illusion of precision than actual accuracy.